.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Test
+on: [ pull_request ]
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    name: Build and Test
+    steps:
+    - name: Checkout this commit
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: C++ Style Check (compiled)
```

Comment:
Where can I see the output of this?

Reply:
The action says it only runs on files added in a specific commit. https://github.com/marketplace/actions/c-style-check-compiled

The output is on the "checks" tab of the PR but there's nothing interesting for this one.

I haven't tested it at all though. I'm expecting a style check could be painful for our codebase but we can try it out.

Reply:
OK, fair enough